Dockerfile.moc-xdmod

```diff
@@ -76,14 +76,15 @@ COPY ./moc_openstack_api_reporting.py /usr/bin/xdmod-openstack-reporting
 COPY ./GetConfigFiles.py /usr/bin/xdmod-get-config-files
 COPY ./xdmod-run-ingestor.sh /usr/bin/xdmod-run-ingestor.sh
 COPY ./xdmod_init.py /usr/bin/xdmod-init 
-RUN chmod g+rwX /usr/bin/xdmod-openstack-hypervisor \
+RUN chmod 774 /usr/bin/xdmod-openstack-hypervisor \
```

Comment:
@rob-baron in this change, you're replacing symbolic modes like `g+rwX` with numeric modes; what problem was this solving?